src/main/java/cs/ualberta/cmput402/tictactoe/scoreboard/Scoreboard.java

```diff
@@ -0,0 +1,47 @@
+package cs.ualberta.cmput402.tictactoe.scoreboard;
+
+public class Scoreboard {
+    private int playerXScore;
+    private int playerOScore;
+    private int tie;
```

Comment:
It might be worthwhile to rename `tie` to `ties`/`tieCount`. `tie` might imply a flag that is set when there is a tie, rather than the tie count.

Reply:
Thank you for your comment. We renamed tie to ties.